tests/libvmruntime/vmruntime.go

```diff
@@ -0,0 +1,37 @@
+package libvmruntime
+
+import (
+	"context"
+
+	"github.com/onsi/ginkgo/v2"
+	"github.com/onsi/gomega"
```

Comment:
usually we use:
```
	. "github.com/onsi/ginkgo/v2"
	. "github.com/onsi/gomega"
```
so that we can simply use `Expect` and `By` without `gomega.Expect` and `ginkgo.By`